Source/control.cpp

```diff
@@ -734,12 +634,146 @@ void DrawSpellList(const Surface &out)
 			y -= SPLICONLENGTH;
 		}
 	}
+
+	return spellListItems;
+}
+
+bool GetSpellListSelection(spell_id &pSpell, spell_type &pSplType)
+{
+	pSpell = spell_id::SPL_INVALID;
+	pSplType = spell_type::RSPLTYPE_INVALID;
+	auto &myPlayer = Players[MyPlayerId];
+
+	for (auto& spellListItem : GetSpellListItems()) {

```

Comment:
clang-format?

Reply:
Fixed 🙂 

Btw:
clang-format is now used in all files or?
Should we run clang-format as validation step in CI? (Fails if clang-format would reformat the code)
Do we want this and if yes, do we have a issue for it?
Cause that would help me to avoid such issues and while reviewing you don't have about format.

Reply:
Yeah I think it would be ok to require parsing clang-format at this point. No issue for it.